domain/usecase/operation.go

```diff
@@ -51,3 +51,24 @@ func (it *Interactor) DoneTask(id int) (*entity.Task, error) {
 	}
 	return task, nil
 }
+
+// 全てのタスクの完了
+func (it *Interactor) AllDoneTask() ([]*entity.Task, error) {
```

Comment:
関数名は動詞で始めてください。DoneAllTasksとかどうでしょうか。

```suggestion
func (it *Interactor) DoneAllTasks() ([]*entity.Task, error) {
```